d3d9ex/d3d9ex.vcxproj

```diff
@@ -132,6 +133,8 @@
       <Optimization>MaxSpeed</Optimization>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
+      <LanguageStandard>stdcpplatest</LanguageStandard>
+      <CreateHotpatchableImage>true</CreateHotpatchableImage>

```

Comment:
Does this help with anything?

Reply:
It's only for x86: https://docs.microsoft.com/en-us/cpp/build/reference/hotpatch-create-hotpatchable-image?view=msvc-160 For all other targets i.e. ARM and x64 compiler behaves same like using this option so i think it's safe.